api/handler/acl.go

```diff
@@ -493,14 +493,18 @@ func (h *handler) PutBucketPolicyHandler(w http.ResponseWriter, r *http.Request)
 }
 
 func parseACLHeaders(header http.Header, key *keys.PublicKey) (*AccessControlPolicy, error) {
```

Comment:
Why keeping the key here?

Reply:
But I think the only thing you do now with this key is getting an account. In which case the key is irrelevant and as S3 never sets any keys into ACLs, why should it care about them during parsing? I do want to make this easier to use in the end, we only have keys in NeoFS itself for compatibility and that can be reconsidered after some transition period.

Reply:
I agree with this point of view. Replaced with `user.ID`